docs/reference/setup/install.asciidoc

```diff
@@ -87,6 +87,10 @@ bundled JVM is treated as an integral part of {es}, which means that Elastic
 takes responsibility for keeping it up to date. Security issues and bugs within
 the bundled JVM are treated as if they were within {es} itself.
 
+You should not use long-running third-party agents that attach to the JVM
+for deep introspection and instrumentation. Such software has been observed
+to reduce Elasticsearch performance, or even freeze or crash nodes.
```

Comment:
I think we should have fewer qualifiers here. It's not just _long-running agents for instrumentation_ that we want to exclude, running with any kind of agent attached is unsupported.

Also I'd prefer stronger wording than "you should not" - ideally "do not" or something equally imperative.